src/actions/profile.action.js

```diff
@@ -58,7 +58,13 @@ const updateProfile = obj => {
       }
       const user = decodeToken();
       const { data } = await http.patch(`/profile/${user.id}`, obj);
-      return dispatch(updateProfileSuccess(data));
+      dispatch(updateProfileSuccess(data));
+      toast.info('You have successfully update your profile.', {
+        type: toast.TYPE.INFO,
+        closeButton: false,
+        position: toast.POSITION.TOP_CENTER,
+      });
+      return 'success';
```

Comment:
You don't need to return anything from an action. Is this return value of `success` going to be used anywhere in the app? @sojida

Reply:
We are not using `success` anywhere. I'll remove it. Thanks